src/tables/ctran_data.py

```diff
@@ -0,0 +1,142 @@
+import sys
+import pandas
+from .table import Table
+from sqlalchemy.exc import SQLAlchemyError
+
+class CTran_Data(Table):
+
+    ###########################################################################
+    # Public Methods
+
+    def __init__(self, user=None, passwd=None, hostname="localhost", db_name="aperature", verbose=False, engine=None):
+        super().__init__(user, passwd, hostname, db_name, verbose, engine)
+        self._schema = "aperature"
+        self._table_name = "ctran_data"
+        self._index_col = "data_row"
+        self._expected_cols = set([
```

Comment:
We should be able to have a row_id (bigserial, I believe) column as input. We'll need to discuss the implications of this, but our sample data, we know now, will have one.